src/jdk.jfr/share/classes/jdk/jfr/internal/dcmd/DCmdCheck.java

```diff
@@ -135,7 +134,7 @@ private void printSettings(Recording recording) {
     private static List<EventType> sortByEventPath(Collection<EventType> events) {
         List<EventType> sorted = new ArrayList<>();
         sorted.addAll(events);
-        Collections.sort(sorted, new Comparator<EventType>() {
+        sorted.sort(new Comparator<EventType>() {
```

Comment:
I wonder if there is a bootstrap issue here (why an anonymous class is used instead of a lambda?)
If a lambda can be used, it cn be simplified to
`sorted.sort(Comparator.comparing(EventType::getName))`

Reply:
As I can see lambdas are used in other places in this module. Replaced